sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +86,36 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    data class RadarMeta(
+        val config: JSONObject?,
+        val remoteTrackingOptions: RadarTrackingOptions?
+    )
+
+    private fun parseMeta(res: JSONObject?): RadarMeta {
+        var meta: JSONObject? = null
+        if (res?.has("meta") == true) {
+            meta = res.getJSONObject("meta")
+        }
```

Comment:
can this be simplified to `val meta = res.optJSONObject("meta")`?

Reply:
ah I like that simplification. I was following the existing pattern in the code, but this is much nicer